pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "stackit-core"
-version = "0.0.1"
+version = "0.0.1a"
```

Comment:
shouldn't it be "0.0.1-alpha"

That's the way to mark pre-releases in semantic versioning

Reply:
Hyphens are not allowed in python versions.
Python internally shortens versions, so "0.0.1a" and "0.0.1alpha" are equal.